pages/animated-stars.css

```diff
@@ -1,52 +1,9 @@
-.animated-stars {
-  width: 100%;
-  height: 40vh;
-}
-
-.nc-main {
-  position: absolute;
-  left: 0;
-  top: 64px;
-  height: 40vh;
-  width: 100vw;
-  overflow-x: hidden;
-  background-size: cover;
-}
-
-.full-wh {
-  position: absolute;
-  top: 0px;
-  left: 0px;
-  bottom: 0px;
-  width: 100%;
-}
-
 .bg-animation {
   position: fixed;
   top: 0;
   left: 0;
-  width: 100%;
-  height: 100%;
-}
-
-.logo-proposal {
-  position: absolute;
-  left: 50%;
-  top: 50%;
-  z-index: 10;
-  width: 500px;
-  transform: translate(-50%, -50%);
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-}
-
-.bg-animation {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
+  width: 100vw;
+  height: 100vh;
```

Comment:
Still thinking whether I like these stars 😪 , I think the best would be to buy a nice illustration from creative market.

Reply:
Yeah the stars are nice enough, we can obviously continue playing around with different things though :+1: